CMakeLists.txt

```diff
@@ -1656,6 +1656,10 @@ if(CONFIG_OUTPUT_STAT)
 endif()
 
 if(CONFIG_BUILD_OUTPUT_STRIPPED)
+if(BOARD MATCHES "^native_posix")
```

Comment:
Two thoughts: 
1. All posix arch boards will have this same issue you mention => this check would need to be on ARCH not on BOARD
2. Having a warning by default will be annoying/confusing for users. if you want to default to y I'd default it to yes only if not POSIX_ARCH.

Reply:
Failures fixed, workaround is gone - resolved.